apps/web/app/globals.css

```
@@ -1,6 +1,5 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+@config "../tailwind.config.ts";
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)